django_components/templatetags/component_tags.py

```diff
@@ -366,6 +605,13 @@ def parse_component_with_args(parser, bits, tag_name):
                 % tag_name
             )
 
+    # component_name: str = component_name.strip("\"'")
+    # if not component_name.isidentifier():
+    #     raise TemplateSyntaxError(
+    #         f"Component name '{component_name}' is not a valid Python "
+    #         f"identifier, e.g. no spaces allowed."
+    #     )
+
```

Comment:
The blocker is that unquotes component names appear to be treated as variables instead of labels. I expand on this above. Resolving so we can focus the discussion in one place